src/pages/Diagnostics.vue

```diff
@@ -1,11 +1,27 @@
 <script lang="ts">
+
 import Vue from 'vue';
+import { mapGetters, mapState } from 'vuex';
 
 import DiagnosticsBody from '@/components/DiagnosticsBody.vue';
+import type { ServerState } from '@/main/commandServer/httpCommandServer';
 
 export default Vue.extend({
   name:       'diagnostics',
   components: { DiagnosticsBody },
+
+  data() {
+    return { rows: [] };
```

Comment:
Is there any need for the rows data prop? It looks to me like the `diagnostics` getter could fulfill all our needs without it.

Reply:
Yes, we don't need it.